src/styles/layout.scss

```diff
@@ -482,4 +483,116 @@ body {
     color: $brand-dark;
     text-align: center;
     font-size: small;
+}
+
+.lockout-overlay {
```

Comment:
you can remove one .lockout-overlay right?

Reply:
Absolutely! good catch